packages/widgets/style/splitpanel.css

```
@@ -18,6 +18,8 @@
 
 .lm-SplitPanel-handle {
   z-index: 1;
+  /* disable size containment as the handle overhangs another div */
+  contain: style!important;
```

```suggestion
  contain: style !important;
```

Unless it's meaningful to have `style!important` as one contiguous token, adding a space would make this more readable.

---

Is using !important here OK because essentially we never want the style to be overwritten by anything?

---

This is a good question. In JupyterLab it would be ok with the logic "we are overriding lumino and changing widget setting to not use strict containment in JS is not worth the trouble". But here we are modifying core lumino so maybe another fix by not adding `contain: strict` would be better. I will need to check if we can do so without breaking API.

---

So instead of adding this style we should just remove this line:

https://github.com/jupyterlab/lumino/blob/133d8725f4dd2c3a56b10d917f1a7b116ff17ce8/packages/widgets/src/splitlayout.ts#L832

Thanks for questioning me - the identification of the problem was right but solution was wrong, another reason not to make PRs at 3am even if those are as simple as 2 lines :)

---

Or rather we should change `strict` to `style` here.

---

No problems! I've searched for '!important' in Lumino before asking and it is used in couple of places in the codebase. Thanks for fix and explanations